### ISSUE TITLE
Get DLQ working

### ISSUE THREAD

**Issue by carderne** [FrontierDevelopmentLab/sat-extractor#10]
@frandorr @Lkruitwagen 

From [here](https://cloud.google.com/pubsub/docs/handling-failures#pubsub_dead_letter_delivery_attempt-python), the service account for PubSub looks like this:

```
PUBSUB_SERVICE_ACCOUNT="service-${project-number}@gcp-sa-pubsub.iam.gserviceaccount.com"
```

i.e. instead of using the service account from the `token.json`.

From CLI:
```
PROJ_NUMBER=$(gcloud projects list \
--filter="$(gcloud config get-value project)" \
--format="value(PROJECT_NUMBER)")

PUBSUB_SERVICE_ACCOUNT="service-$PROJ_NUMBER@gcp-sa-pubsub.iam.gserviceaccount.com"
```

And then bind the account as already done:
```
gcloud pubsub topics add-iam-policy-binding "$DLQ_TOPIC" \
  --member="serviceAccount:$PUBSUB_SERVICE_ACCOUNT" \
  --role=roles/pubsub.publisher

gcloud pubsub subscriptions add-iam-policy-binding "$MAIN_SUBSCRIPTION" \
  --member="serviceAccount:$PUBSUB_SERVICE_ACCOUNT" \
  --role=roles/pubsub.subscriber
```

**Comment by carderne** [comment]
I realised probably the main thing wrong in the code is actually that you don't create a DLQ subscription...
eg
```
gcloud pubsub subscriptions create "stacextractor-dlq-sub-name" \
  --topic="projects/$PROJECT_ID/topics/$DLQ_TOPIC"
```

**Comment by carderne** [comment]
Fixed in #11
